wiki/en/Running-a-Server.md

```diff
@@ -290,17 +259,17 @@ It is highly recommended to test your Server on a **Public Directory** first so
 
```

Comment:
Line 256 (new) needs to be a level 2 heading.

Reply:
Fixed.